src/components/Display.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result }) => (
+  <div>{ result }</div>
+);
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
```

Comment:
Let's give `Display` component a default prop of string `0`. See https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values

> its result prop should be a String with defualt value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))